src/CacheManager.php

```diff
@@ -18,7 +18,7 @@ class CacheManager extends BaseCacheManager
      */
     public function __call($method, $parameters)
     {
-        $tags = [config('tenancy.cache.tag_base') . tenant()->getTenantKey()];
+        $tags = [config('tenancy.cache.tag_base') . tenant()?->getTenantKey()];
```

Comment:
Any reason for this change?

Reply:
I don't think there was a reason for this, let me check

Reply:
Yeah, tests run fine and I can't think of a reason for keeping this change. Reverted this